libcuda/cuda_runtime_api.cc

```diff
@@ -1121,10 +1121,10 @@ __host__ cudaError_t CUDARTAPI cudaDeviceGetAttribute(int *value, enum cudaDevic
                         *value= 0;
                         break;
                 case 75:
-                        *value= 9 ;
+                        *value= 7 ;
```

Comment:
Need a comment explaining what this is for + need to add to config file and get from there.